src/components/wasd-controls.js

```diff
@@ -98,21 +101,29 @@ module.exports.Component = registerComponent('wasd-controls', {
 
   attachEventListeners: function () {
     // Keyboard events
-    window.addEventListener('keydown', this.onKeyDown, false);
-    window.addEventListener('keyup', this.onKeyUp, false);
+    window.addEventListener('blur', this.onBlur);
```

Comment:
oh, very good call.

hmm, the more I think about it we probably shouldn't call `pause` on `blur` because that [removes the event listeners](https://github.com/aframevr/aframe/pull/1211/files#diff-6231789e7e7fac0fbea47834d09b3bdcR91). and then we'd have to reattach them. seems like maybe a waste. thoughts?